packages/react-native-editor/package.json

```diff
@@ -30,6 +30,7 @@
 	"react-native": "src/index",
 	"dependencies": {
 		"@babel/runtime": "^7.13.10",
+		"@react-native-clipboard/clipboard": "1.9.0",
```

Comment:
**Curious:** Did you pivot to `@react-native-clipboard/clipboard` preemptively, or did `0.66` finally remove the API and replace it with a stub?

Reply:
I do not believe it is removed in `0.66`. I merely chose to address it along with other deprecations warnings in an effort to clean up the logs some.